README.md

```diff
@@ -15,6 +15,18 @@ Current collection
     - ansible-lint latest (5.2.x)
       - with "[community,yamllint]"
   - config: strictly in-repo
+- [doc8][]
+  > NOTE: ignores Sphinx-specific false positives
+  >
+  > TLDR; this isn't really a Sphinx linter
+  - docker
+  - Fedora 35
+  - Py 3, Fedora default (3.10.x)
+  - pip:
+    - doc8==0.9.1
+  - config
+    - defaults
+    - 'path' parameter, default 'docs/'
```

Comment:
Like #10, do we want to distinguish between config and inputs?

Reply:
ACK